janus_server/src/bin/janus_cli.rs

```diff
@@ -280,7 +282,7 @@ struct KubernetesSecretOptions {
         env = "DATASTORE_KEYS_SECRET_KEY",
         takes_value = true,
         help = "Key into data of datastore key Kubernetes secret",
-        default_value = "datastore_key"
+        default_value = "datastore_keys"
```

Comment:
Same issue: I think this default value is incorrect. Our deployment includes `DATASTORE_KEYS: <set to the key 'datastore_key' in secret 'datastore-key'> Optional: true`, and the Janus component binaries read the environment variable `DATASTORE_KEYS` via clap.